### PR TITLE
remove menu items from topnav, fix cog size and dropdown

### PR DESCRIPTION
**Pull request by rkassa**
- remove menu items from topnav
- keep settings cog; fix cog size
- fix settings cog dropdown styles

### CODE REVIEW

File: deploy [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!//Library/Frameworks/Python.framework/Versions/3.8/bin/python3
```
Comment (lmanul) [inline deploy:1]: Can you revert this?

File: index.html [hunk 2]
```diff
@@ -11,7 +11,8 @@
     <div id="modal"></div>
   </div>
   <div id="topbar-container"><div id="topbar"></div></div>
-  <div id="app"></div>
+  <div id="app"><div id="sidebar"></div></div>
```
Comment (lmanul) [inline index.html:14]: Why this change? Not all views require a side bar.

File: js/mapview.js [hunk 3]
```diff
@@ -58,9 +58,9 @@ fetchData() {
 render() {
   super.render();
   let app = document.getElementById('app');
-  app.innerHTML = '';
-  let sideBarEl = document.createElement('div');
-  sideBarEl.setAttribute('id', 'sidebar');
+  // app.innerHTML = '';
```
Comment (lmanul) [inline js/mapview.js:61]: In general, why comment stuff out instead of removing?

Comment (lmanul) [inline js/mapview.js:61]: It seems like this would completely prevent switching views without having leftovers of the previous view?

**Comment by rkassa** [comment]
@lmanul i'm going to close this PR and open a separate one for the small gear/cog and dropdown fix. will leave the sidebar and main topnav untouched :)
